conf/base/catalog.yml

```
@@ -108,3 +112,11 @@ works_for_concepts:
 nih.data_collection.clinical_trials.raw:
   <<: *_csv_ptd
   path: s3://alphafold-impact/data/01_raw/nih/clinical_trials/
+
+nih.processing.clinical_trials_with_references.intermediate:
```

We probably want to keep naming conventions as standardised as possible, so I'd suggest changing these two:
```suggestion
nih.data_processing.clinical_trials_with_references.intermediate:
```